### ISSUE TITLE
Interactive Jupyter Cell -> no keyboard input possible

### ISSUE THREAD

**Issue by michaelschwier** [microsoft/vscode-jupyter#4087]
## Environment data

-   VS Code version: 1.51.1
-   Jupyter Extension version (available under the Extensions sidebar): 2020.11.392013122
-   Python Extension version (available under the Extensions sidebar): 2020.11.371526539
-   OS (Windows | Mac | Linux distro) and version: Win 10 Version 10.0.17763 Build 17763
-   Python and/or Anaconda version: 3.7.7
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local 

## Expected behaviour

Keyboard input in interactive input field should work

## Actual behaviour

Keyboard input in interactive input field not possible

## Steps to reproduce:

Since the latest update of vscode (1.51.1), when I run jupyter cells and use the interactive input field, after some time no keyboard input is possible anymore.

Steps to reproduce:

Define a few jupyter-like code cells (in my case they also plot smth with seaborn/matplotlib)
Use the interactive command input to inspect some of your variables
Repeat the above steps a few times, at some point the interactive command input does not recognize keyboard input anymore, restart needed

I got forwarded from [vscode issues](https://github.com/microsoft/vscode/issues/110526) to here.

**Comment by rchiodo** [comment]
Hmm. We can no longer repro this after the fix for #4001 went through (which you should have based on your extension versions)

Can you click anywhere in the window? Or is just the input box at the bottom frozen?

**Comment by rchiodo** [comment]
Would you be able to capture a gif of it not working? It would be helpful to see where the focus moves (the original bug was a focus problem).

This is possibly related to this bug as well:
https://github.com/microsoft/vscode/issues/97992

When I made the fix for #446, the interactive window would think it had focus but it actually didn't. So when we attempted to give it focus, it would ignore the focus request. Generally you could work around the issue by maximizing/minimizing VS code or doing something that would cause a refresh of the windows.

**Comment by michaelschwier** [comment]
Hey, thanks for the quick reply. I updated vscode to 1.51.1 and tested it again and indeed cannot reproduce. I'll keep monitoring.

I think my mistake was this: When I filed the initial issue at vscode's git 3 weeks ago I disabled auto updates and manually downgraded to vscode 1.50.1 to be able to work ... but I did not realize that auto updates for extensions are an independent setting and were still active. So now when the vscode guys forwarded me to here, I just looked up the extension version, assuming it is still the one from 3 weeks ago - but I guess the extension was updated in the meantime.

Sorry for the false alarm, closing this.
